torch/_inductor/lowering.py

```diff
@@ -3621,6 +3621,9 @@ def _validate_reduction_axis(x, axis):
         axis = [axis]
     elif not axis:
         axis = range(len(size))
+    if len(size) == 0:
+        assert tuple(axis) in [(), (0,), (-1,)], f"invalid axis: {axis}"
+        return []
```

Comment:
This doesn't actually do a reduction, does it? So it would be wrong for e.g. `argmin` where a single item reduction isn't the same as a copy.

Reply:
I didn't test argmin, it is worth checking.